common/common.cpp

```diff
@@ -2089,6 +2098,12 @@ struct llama_model * llama_load_model_from_url(
         return NULL;
     }
 
+    if(params.hf_token!=NULL){
```

Comment:
Mind your linter. Spaces are important

Reply:
how do I lint this code ? should I just use the linter in github workflows installed?

Reply:
Which IDE are you using ?